requirements.txt

```diff
@@ -1,6 +1,7 @@
 beautifulsoup4~=4.9.1
 binarytree==5.1.0
 canvasapi==2.0.0
+discord==1.0.1
```

Comment:
wrong discord lol

Reply:
PyCharm told me to add it -- however, PyCharm does say some strange things.

![Discord](https://user-images.githubusercontent.com/25561432/103988419-f67cd680-5142-11eb-9ab4-43a64c944db9.PNG)

Reply:
I'll just remove it.